### DISCUSSION TITLE
Can /runner and /runner/_work be mounted using a PVC instead of emptyDir?

### DISCUSSION THREAD

**Discussion by LogeshGovi** [actions/actions-runner-controller#1113] [Questions]
Hi I am using runnerset for a project. I would like to know if /runner and /runner/_work paths can be mounted from an existing PVC or are they always intended to be mounted using emptyDir ephemeral storage? Info in this regard would be greatly appreciated

**Comment by shichengripple001** [comment]
you can set workDir and mount PVC
